ext/reflection/php_reflection.c

```diff
@@ -4422,12 +4422,17 @@ ZEND_METHOD(ReflectionClass, getConstants)
 	zend_string *key;
 	zend_class_constant *constant;
 	zval val;
-	zend_long filter = ZEND_ACC_PPP_MASK;
+	zend_long filter;
+	zend_bool filter_is_null = 1;
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS(), "|l", &filter) == FAILURE) {
+	if (zend_parse_parameters(ZEND_NUM_ARGS(), "|l!", &filter, &filter_is_null) == FAILURE) {
```

Comment:
I made these params nullable because the previous default value was horrible, there were precedents for doing so before (e.g. `ReflectionClass:getMethods()`)